recipes/qp/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "qp" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
I don't think you need this line.

Reply:
No sure if  you mean line 6 or line 9.    I tried removing line 6 and it fails, (see below).

Reply:
Ah, you need it for the SCM stuff. Ok, no probs then!